keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -459,87 +461,120 @@ Windows privileged containers will be implemented with [Job Objects](https://doc
+- Directory mounts (i.e. directory C:\test mapped to C:\test in the privileged container) are still being investigated. Note that file system mapping in the current implementation would expose both the host filesystem and the container filesystem to the processes in the privileged container. We are still investigation how external mounts such as secrets or storage accounts would be added. The default visibility of all the host filesystem may present backwards compatibility challenges in the future if host path mounting is enabled in Windows. Currently the only method to restrict would be via the user in a way similar to [UID or GID](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) in the PSP. Currently however, this host path mounting is unlikely to be enabled in the future as a restriction of Job Objects.
+
 #### Container Images
-- A slim base image may be required to satisfy hierarchy requirements from HCS. It was found that the graphdriver calls expect certain files to be in a windows image when un-tarring. These files were found to be simply several registry hive files in /windows/system32/config (which can be empty).
-- We are currently targeting to release a simple image alongside this feature including the above mentioned files, which can be used instead of traditional Windows base images of server core or otherwise. Server core and others that container the necessary files would also work as privileged container base images. 
+
+- Privileged containers can be built on top of existing Windows base iamges (nanoserver, servcore, etc).
+- A slim base image may be introduced to act as a replacement for [scratch]. On windows graphdriver calls expect some files (mainly registry hives) and these would be included in the slim base image.
+- Privileged containers will not inherit the same same [compatability requriments](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility) as process isolated containers from an OS perspecitve.
+
 #### Container Image Build/Definition
-- This is another ongoing area of investigation and open to feedback. 
- 
+
+- This is another ongoing area of investigation and open to feedback. Currently only a subset of dockerfile operations are supported (ADD, COPY, PATH, ENTRYPOINT, etc).
```

Comment:
For the trimmed down image, anything that requires an operation to occur inside a running container (so RUN is axed for starters).

Reply:
Need feedback from @dcantah here. I think RUN commands generally will not work because they can modify system state (registry, services, etc) and those changes won't be persist in the privileged container case.

Example: I'm thinking of a case where a user makes a docker file that installs a windows service and expects that to run when the contianer starts. What would happen in the privileged contianer case?

Reply:
@dcantah @jsturtevant, should we list which  operations **won't** work our list operations that **will** work here?

Reply:
The trimmed down image isn't even going to be available for the Alpha is it? We haven't talked to MCR to see about hosting it yet, so is it worth going into detail on build instructions for it?

Reply:
I think which ones WILL work seems more appropriate though

Reply:
I want to call our that not all commands won't work. I'll just say we're continuing to investigate

Reply:
Actually, i think me saying this is an ongoing area of investigation is sufficient for now.

Reply:
Sounds good!